web/satellite/src/components/account/billing/paymentMethods/PaymentMethods.vue

```diff
@@ -38,6 +38,7 @@
                 @toggleIsLoading="toggleIsLoading"
                 @cancel="onCancel"
             />
+            <!-- Use this component -->
```

Comment:
is this comment still useful? I'm not sure if you added it for your own reference while building this, or if you are adding clarification for future readers. If it's to add clarification, I'm not sure how to interpret the comment

Reply:
removed